### DISCUSSION TITLE
Delete specific files or folders

### DISCUSSION THREAD

**Discussion by gkagg** [filebrowser/filebrowser#1479] [Q&A]
Hello. 

Congratulations on your very good program!

I have one question. Can some user have the permission to delete only specific files or folders and it's contents?
Let's say we have the root folder containing 3 folders. Folder One, Folder Two, and Folder Three. Can a user have the right to delete files only in Folder Two and Folder Three?

Regards
George

**Comment by o1egl** [comment]
No. This is not possible to do in filebrowser
